feature/bgp/policybase/otg_tests/nested_policies/metadata.textproto

```diff
@@ -0,0 +1,20 @@
+# proto-file: third_party/openconfig/featureprofiles/proto/metadata.proto
```

Comment:
Update the path here to `github.com/openconfig/featureprofiles/proto/metadata.proto`

Reply:
not required file - removed